mathics/builtin/comparison.py

```diff
@@ -1024,6 +1035,7 @@ class Max(_MinMax):
      = x
     """
 
+    summary_text = "maximum value in a list"
```

Comment:
I'm not sure if this is correct,  "maximum number of list" and "biggest number in a list" sounds more natural.  @rocky ?

Reply:
Personally, I like "largest" or "biggest" over "maximum" on the idea that I like a simple word when a simple word fits. 

However for number versus value, I do see a problem: `Max` handles things which aren't numbers like lists:

```
In[2]:= Max[{0, 1}, {0, 5}]
Out[2]= 5
```



But  I should note that we have an error in Max (and probably Min

```
In[1]:= Max["a", "b"]
Traceback (most recent call last):
  File "/tmp/bin/mathics", line 33, in <module>
    sys.exit(load_entry_point('Mathics3', 'console_scripts', 'mathics')())
  File "/tmp/Mathics3/mathics-core/mathics/main.py", line 444, in main
    result = evaluation.evaluate(query, timeout=settings.TIMEOUT)
  File "/tmp/Mathics3/mathics-core/mathics/core/evaluation.py", line 343, in evaluate
    result = run_with_timeout_and_stack(evaluate, timeout, self)
  File "/tmp/Mathics3/mathics-core/mathics/core/evaluation.py", line 98, in run_with_timeout_and_stack
    return request()
  File "/tmp/Mathics3/mathics-core/mathics/core/evaluation.py", line 314, in evaluate
    self.last_eval = query.evaluate(self)
  File "/tmp/Mathics3/mathics-core/mathics/core/expression.py", line 658, in evaluate
    expr, reevaluate = expr.rewrite_apply_eval_step(evaluation)
  File "/tmp/Mathics3/mathics-core/mathics/core/expression.py", line 1316, in rewrite_apply_eval_step
    result = rule.apply(new, evaluation, fully=False)
  File "/tmp/Mathics3/mathics-core/mathics/core/rules.py", line 79, in apply
    self.pattern.match(yield_match, expression, {}, evaluation, fully=fully)
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 344, in match
    self.head.match(yield_head, expression.get_head(), vars, evaluation)
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 188, in match_symbol
    yield_func(vars, None)
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 334, in yield_head
    self.get_pre_choices(
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 480, in get_pre_choices
    per_name(yield_func, list(groups.items()), vars)
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 475, in per_name
    yield_name(vars)
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 311, in yield_choice
    self.match_leaf(
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 693, in match_leaf
    self.get_wrappings(
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 516, in get_wrappings
    yield_func(sequence)
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 681, in yield_wrapping
    leaf.match(
  File "/tmp/Mathics3/mathics-core/mathics/builtin/patterns.py", line 1000, in match
    self.pattern.match(yield_func, expression, new_vars, evaluation)
  File "/tmp/Mathics3/mathics-core/mathics/builtin/patterns.py", line 1315, in match
    yield_func(vars, None)
  File "/tmp/Mathics3/mathics-core/mathics/core/pattern.py", line 678, in match_yield
    yield_func(new_vars, items_rest)
  File "/tmp/Mathics3/mathics-core/mathics/core/rules.py", line 54, in yield_match
    new_expression = self.do_replace(expression, vars, options, evaluation)
  File "/tmp/Mathics3/mathics-core/mathics/core/rules.py", line 211, in do_replace
    return self.function(evaluation=evaluation, **vars_noctx)
  File "/tmp/Mathics3/mathics-core/mathics/builtin/comparison.py", line 976, in apply
    c = do_cmp(leaf, best)
  File "/tmp/Mathics3/mathics-core/mathics/builtin/comparison.py", line 517, in do_cmp
    if s1.is_Float and s2.is_Float:
AttributeError: 'NoneType' object has no attribute 'is_Float'
```

Reply:
Yes, there is a problem with string comparisons. `"a"<"b"`  also raises the same error.

Reply:
So something like "get the largest number" would fit here, "largest number of a list" don't fit as it can be 1 or more lists.

Reply:
I fixed the bug and changed the usage_text. @rocky, @TiagoCavalcante, check it when you have time.